test/index.md

```diff
@@ -20,6 +20,10 @@ title: Testing GhostText 👻
 		height: 300px;
 		max-width: 80% !important;
 	}
+	#monaco-editor {
+		width: 100%;
+		height: 100%;
```

Comment:
The heigh should be the same as others, not "100%" (which doesn't do anything here I think)